tox.ini

```diff
@@ -25,60 +25,29 @@ passenv =
 [testenv:fmt]
 description = Apply coding style standards to code
 deps =
-    black
-    ruff
+    -r{toxinidir}/fmt-requirements.txt
 commands =
+    isort {[vars]all_path}
     black {[vars]all_path}
-    ruff --fix {[vars]all_path}
 
 [testenv:lint]
 description = Check code against coding style standards
 deps =
-    black
-    ruff
-    codespell
+    -r{toxinidir}/lint-requirements.txt
 commands =
-    # uncomment the following line if this charm owns a lib
-    # codespell {[vars]lib_path}
-    codespell {toxinidir} \
-              --skip {toxinidir}/.git \
-              --skip {toxinidir}/.tox \
-              --skip {toxinidir}/build \
-              --skip {toxinidir}/lib \
-              --skip {toxinidir}/venv \
-              --skip {toxinidir}/.mypy_cache \
-              --skip {toxinidir}/icon.svg
-    
-    ruff {[vars]all_path}
+    codespell {toxinidir}/ --skip {toxinidir}/.git --skip {toxinidir}/.tox \
+      --skip {toxinidir}/build --skip {toxinidir}/lib --skip {toxinidir}/venv \
+      --skip {toxinidir}/.venv --skip {toxinidir}/.mypy_cache --skip {toxinidir}/icon.svg
+    # pflake8 wrapper supports config from pyproject.toml
+    pflake8 {[vars]all_path}
+    isort --check-only --diff {[vars]all_path}
     black --check --diff {[vars]all_path}
 
 [testenv:unit]
 description = Run unit tests
 deps =
-    pytest
-    coverage[toml]
-    -r{toxinidir}/requirements.txt
+    -r{toxinidir}/unit-requirements.txt
 commands =
     coverage run --source={[vars]src_path} \
-                 -m pytest \
-                 --ignore={[vars]tst_path}integration \
-                 --tb native \
-                 -v \
-                 -s \
-                 {posargs}
+        -m pytest --ignore={[vars]tst_path}integration -vv --tb native -s {posargs}
     coverage report
-
-[testenv:integration]
```

Comment:
add tox env for integration tests:
```ini
[testenv:integration]
description = Run integration tests
deps =
    -r{toxinidir}/integration-requirements.txt
commands =
    pytest -v --tb native {[vars]tst_path}integration --log-cli-level=INFO -s {posargs}
```

Reply:
I'm not actually done with the integration tests. I'd like to add them when I'm finished with the relations.

Reply:
I forgot to remove the template integration test from the PR